src/PrestaShopBundle/Form/Admin/Sell/Order/Delivery/SlipOptionsType.php

```diff
@@ -49,15 +54,60 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 TranslatableType::class,
                 [
                     'type' => TextType::class,
+                    'label' => $this->trans('Delivery prefix', 'Admin.Orderscustomers.Feature'),
+                    'help' => $this->trans('Prefix used for delivery slips.', 'Admin.Orderscustomers.Help') . ' ' .
+                        $this->trans('Invalid characters:', 'Admin.Notifications.Info') . ' <',
+                    'multistore_configuration_key' => 'PS_DELIVERY_PREFIX',
+                    'options' => [
+                        'constraints' => [
+                            new Regex([
+                                'pattern' => '[\<]',
+                                'match' => false,
+                                'message' => $this->trans(
+                                    'The %s character is invalid',
```

Comment:
```suggestion
                                    'The %s character is invalid.',
```
Should we add the plural form too in case there is more than one invalid character?